### PR TITLE
Bulk arrayvalue test

### PR DESCRIPTION
**Pull request by anubhav-sharma13**
In the previous iteration, the test case has been updated in ```DocStoreTest``` class, so updated the corresponding ```MongoDocStoreTest``` Class.

### CODE REVIEW

**Comment by anubhav-sharma13** [comment]
@kotharironak please approve this for merging.

File: document-store/src/integrationTest/java/org/hypertrace/core/documentstore/mongo/MongoDocStoreTest.java [hunk 1]
```diff
@@ -1,6 +1,5 @@
 package org.hypertrace.core.documentstore.mongo;
```
Comment (kotharironak) [inline document-store/src/integrationTest/java/org/hypertrace/core/documentstore/mongo/MongoDocStoreTest.java:1]: Why is this removed?

**Review by kotharironak — APPROVED** [review]
